app/src/cli/mod.rs

```diff
@@ -66,4 +70,8 @@ enum Command {
     Lsp(lsp::Args),
     /// Lift local (co)matches of a type to the top-level
     Lift(lift::Args),
+    /// Generate documentation for a file
+    Doc(doc::Args),
+    /// Clean target_pol directory
```

Comment:
```suggestion
    /// Remove all generated files
```
If you hardcode the name `target_pol` here then it becomes more difficult to find the reference if we change the name of the directory later.